test/src/module/command/backupTest.c

```diff
@@ -3294,9 +3295,24 @@ testRun(void)
             hrnCfgArgRawZ(argList, cfgOptRepoRetentionFull, "1");
             hrnCfgArgRawStrId(argList, cfgOptType, backupTypeIncr);
             hrnCfgArgRawBool(argList, cfgOptDelta, true);
+            hrnCfgArgRawBool(argList, cfgOptPageHeaderCheck, false);
             hrnCfgArgRawBool(argList, cfgOptRepoHardlink, true);
             HRN_CFG_LOAD(cfgCmdBackup, argList);
 
+            // File with bad page checksum and header errors that will be ignored
+            Buffer *relation = bufNew(PG_PAGE_SIZE_DEFAULT * 3);
+            memset(bufPtr(relation), 0, bufSize(relation));
+            *(PageHeaderData *)(bufPtr(relation) + (PG_PAGE_SIZE_DEFAULT * 0x00)) = (PageHeaderData){.pd_upper = 0xFF};
+            *(PageHeaderData *)(bufPtr(relation) + (PG_PAGE_SIZE_DEFAULT * 0x01)) = (PageHeaderData){.pd_upper = 0x00};
+            *(PageHeaderData *)(bufPtr(relation) + (PG_PAGE_SIZE_DEFAULT * 0x02)) = (PageHeaderData){.pd_upper = 0x00};
+            (bufPtr(relation) + (PG_PAGE_SIZE_DEFAULT * 0x02))[PG_PAGE_SIZE_DEFAULT - 1] = 0xFF;
+            ((PageHeaderData *)(bufPtr(relation) + (PG_PAGE_SIZE_DEFAULT * 0x02)))->pd_checksum = pgPageChecksum(
```

Comment:
(though arguably at least the first suggested one is probably covered in the other checksum tests)

Reply:
The checksum is being set in page 2. The invalid checksum (from the comment) is in page 0.

> (though arguably at least the first suggested one is probably covered in the other checksum tests)

True, but I want to see at least one error here so we know the code is running. Otherwise these checks could be skipped entirely and the test would pass.

Reply:
Would like to see the "pd_upper 0, invalid non-zero checksum" case as well, if it's trivial to add at this point.  I'd assume setting the checksum + 1 would be sufficient, or calculating a checksum then incrementing a different page byte would be enough.

Reply:
Sound good. Add in cbd01a803.

Reply:
LGTM